scripts/freeze.mjs

```diff
@@ -0,0 +1,111 @@
+/**
+ * Code-freeze helper script.
+ * Required to be run on a release branch.
+ *
+ * It takes extends the committed changes with a new version and release date, runs builds and tests and pushes the
+ * changes to the release branch.
+ */
+import { createRequire } from 'module';
+import inquirer from 'inquirer';
+import {
+  cleanNodeModules,
+  displayErrorMessage, scheduleRelease,
+  spawnProcess, verifyBundles
+// eslint-disable-next-line import/extensions
+} from './utils/index.mjs';
+
+// Using `require` exclusively to import `json` files. Prevents 'experimental' warnings from being thrown in the
+// console.
+const require = createRequire(import.meta.url);
+const [version, releaseDate] = process.argv.slice(2);
+
+// Initial verification prompt
+inquirer.prompt([
+  {
+    type: 'confirm',
+    name: 'continueFreeze',
+    message: 'This script will create and commit a code-freeze of the entire repo. \nContinue?',
+    default: true,
+  }
+]).then(async(answers) => {
+  if (!answers.continueFreeze) {
+    return;
+  }
+
+  // Check if we're using the minimum required npm version.
+  await spawnProcess('npm --version', true, (childProcess) => {
+    const npmVersion = childProcess.stdout.toString().replace('\n', '').split('.');
```

Comment:
```suggestion
    const npmVersion = childProcess.stdout.toString()
```
The execa already strips the final newline character https://www.npmjs.com/package/execa#stripfinalnewline.